web/static/js/components/room.jsx

```diff
@@ -13,7 +13,9 @@ class Room extends Component {
     super(props)
     this.state = { ideas: [], showActionItem: false }
     this.handleIdeaSubmission = this.handleIdeaSubmission.bind(this)
+    this.handleIdeaDeletion = this.handleIdeaDeletion.bind(this)
     this.handleToggleActionItem = this.handleToggleActionItem.bind(this)
+    this._removeIdea = this._removeIdea.bind(this)
```

Comment:
might want to migrate to [class property initializers](https://babeljs.io/docs/plugins/transform-class-properties/), .e.g.,

```javascript
class ThisComponent {
  _removeIdea = () => {
    // sweet code here
  };
}

makes binding less a chore. can also move `state` initialization to an initializer and completely scrap the constructor. [proof](https://twitter.com/ianmcnally_/status/780523739242962944)